pkg/k8s/apis/cilium.io/client/crds/v2/ciliumclusterwidenetworkpolicies.yaml

```diff
@@ -1545,6 +1545,28 @@ spec:
                       type: array
                   type: object
                 type: array
+              enableDefaultDeny:
+                description: "EnableDefaultDeny determines whether this policy configures
+                  the subject endpoint(s) to have a default deny mode. If enabled,
+                  this causes all traffic not explicitly allowed by a network policy
+                  to be dropped. \n If not specified, the default is true for each
```

Comment:
I don't see @learnitall's feedback fully addressed here.

Reply:
I mostly incorporated it, is there something you see missing?

Reply:
Looks like my review might have been stale; looks good.